### PR TITLE
Added unload, load commands and PL translation

### PR DESCRIPTION
**Pull request by p0358**
Hope that code isn't that bad 😆 It works at least.

### CODE REVIEW

File: src/commands/load.js
Comment (drkno) [inline src/commands/load.js]: Mix of ' ' and '\t' spacing. For consistency with the main Concierge project and the rest of this repo, 4 space indentation should be used.

(On a personal note: I don't really care which indentation type is used tbh, as long as it is consistent)

**Review by drkno — CHANGES_REQUESTED** [review]
Has it been:
- manually tested on valid module names
- manually tested on invalid module names

Please increment the version number to `3.3`

**Review by drkno — CHANGES_REQUESTED** [review]
- You are currently able to `load` a module that is already loaded
- It throws an exception trying to `load` a module that does not exist
- It throws an exception trying to `unload` a module that has already been unloaded
- Any module that has been installed through `kpm`, requires `kpm_` to be put in front of its name in order to load it. Note: this is not an important issue to fix.

File: src/commands/load.js
Comment (p0358) [reply src/commands/load.js]: About that spacing - I have my Notepad++ set to make 4 space intendation. These tab spacing was there already, I copied that code fragment you quoted here from reload.js. I can correct that of course alone with other issues, will do that tomorrow.

Comment (drkno) [reply src/commands/load.js]: Hmmmm, interesting. Maybe reload.js missed out on the gigantic consistency refactor. Regardless it should be consistent. 
Aside from the things mentioned, good work. I can forsee what you have done being very useful :) (sorry, I was reminded that I can be very blunt in my pull requests)

Comment (p0358) [reply src/commands/load.js]: All right, I will add another commit which will fix spacing, increment version number.
In that commit:
- It doesn't throw an exception when you try to `unload' a module that has already been unloaded or doesn't exist
- It doesn't let you load a module that is already loaded
- It still throws an exception when you try to load non-existing module (exception error is `ENOENT: no such file or directory`) - load.js has `try` block, so it seems like it's a problem with `verifyModule` function, I bet it has something like this:

``` javascript
            try {
                // do something
            }
            catch (e) {
                console.critical(e);
            }
```

and that's why we see an error in the console. Hmm, maybe it could be fixed within KPM by adding some check (if the dir exists).
I can try to add ability to load modules with `kpm_` prefix without needing to type that prefix, but I don't think that's really needed, it's more for manual modules loading, if someone's doing some testing on KPM module, he can write kpm_modulename then.
Another issue - when I restart Concierge, it rewrites language files, and after that rewrite, all Polish chars are broken... (it doesn't break the main Concierge translations though, only KPM ones [might do that for other modules too, untested]). Maybe it's bad coding (not UTF-8?) once again, but it displays OK, until restart when Concierge rewrites the file.

**Comment by p0358** [comment]
Commit added.
- Translations seems to be fixed after changing coding to UTF-8 - we can forget about it except that i18n files must be in UTF-8.
- All issues fixed - directory not exists error resolved by adding a check for that before calling `verifyModule` function.
- Incremented version number.
- Changed spacing in all files inside `src` directory.

@mrkno You can test if everything works correctly and if the code looks OK I can merge it (I think I can, because I see that green "Merge pull request" button now 😄).
[sorry for all the gramatical mistakes, I just don't read what I wrote before sending it 😄]

**Review by drkno — APPROVED** [review]

**Comment by drkno** [comment]
@p0358 All the manual tests I did passed :). As a Concierge member, you should have write access to all of the repos in the organization now (so you can git branch rather than fork and create new repos. For all but the main modules like this one PR's arn't really necessary). I'll let you press the green merge button if you are satisfied with what you have.

**Pull request merged**
